watchdog-config/extra-sql-queries.yml

```diff
@@ -75,3 +75,400 @@ users_with_insificient_disk:
+        usage: "LABEL"
+
+manual_feedback_docs:
+  query: |
+    SELECT
+        feedback.partner_name,
+        reported::varchar(255),
+        detail,
+        count(*) AS detail_count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_feedback_docs AS feedback
+    LEFT JOIN
+        monitoring_urls ON feedback.partner_name=monitoring_urls.partner_name
+    WHERE source = 'manual'
+    GROUP BY
+        feedback.partner_name,
+        reported,
+        source,
+        detail,
+        cht_instance
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - detail:
+        usage: "LABEL"
+    - detail_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+automatic_feedback_docs:
+  query: |
+    SELECT
+        feedback.partner_name,
+        reported::varchar(255),
+        detail,
+        count(*) AS detail_count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_feedback_docs AS feedback
+    LEFT JOIN
+        monitoring_urls ON feedback.partner_name=monitoring_urls.partner_name
+    WHERE source = 'automatic'
+    GROUP BY
+        feedback.partner_name,
+        reported,
+        source,
+        detail,
+        cht_instance
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - detail:
+        usage: "LABEL"
+    - detail_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+replication_by_status:
+  query: |
+    SELECT
+        status.partner_name,
+        period_start::varchar(255),
+        replication_success_count,
+        replication_failure_count,
+        replication_denied_count,
+        replication_error_count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_replication_by_status AS status
+    LEFT JOIN
+        monitoring_urls ON status.partner_name=monitoring_urls.partner_name
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - period_start:
+        usage: "LABEL"
+    - replication_success_count:
+        usage: "GAUGE"
+    - replication_failure_count:
+        usage: "GAUGE"
+    - replication_denied_count:
+        usage: "GAUGE"
+    - replication_error_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+# replication_activity:
+#   query: |
+#     SELECT
+#         sms.partner_name,
+#         reported::varchar(255),
+#         delta_of_doc AS delta_docs,
+#         replace(monitoring_urls.url, 'https://', '') as cht_instance
+#     FROM app_monitoring_sms_error_and_users_docs_replication AS sms
+#     LEFT JOIN
+#         monitoring_urls ON sms.partner_name=monitoring_urls.partner_name
+#   metrics:
+#     - partner_name:
+#         usage: "LABEL"
+#     - reported:
+#         usage: "LABEL"
+#     - delta_docs:
+#         usage: "GAUGE"
+#     - cht_instance:
+#         usage: "LABEL"
```

Comment:
no need to store this here!  

```suggestion
```

Reply:
Sorry I didn't add a comment earlier on this. The query was causing 500 errors within SQL exporter and I couldn't figure out why. It's required for the docs synced per day metric and runs well on the PG client. No useful pointers in container logs. I could use your help with diagnosing why the 500s are happening.

Reply:
hmmm - hard for me to say without being able to reproduce it.   I did play around with the query and see that as is it's returning 21k results and this only returns ~800 which seems more sane. Maybe helpful as a starting point to debug by reducing data it's processing?

```sql
SELECT
	sms.partner_name,
	reported::date,
	delta_of_doc AS delta_docs,
	replace(monitoring_urls.url, 'https://', '') as cht_instance
FROM 
	app_monitoring_sms_error_and_users_docs_replication AS sms
LEFT JOIN
  monitoring_urls ON sms.partner_name = monitoring_urls.partner_name
WHERE 
	monitoring_urls.url is not NULL and
	delta_of_doc > 0 and
	reported > current_date - interval '30' day;
```

Reply:
Reducing data processing worked, thank you

Reply:
new query looks great and is fast!